apps/ledger-live-desktop/src/renderer/hooks/useDeeplinking.ts

```diff
@@ -82,6 +82,18 @@ export function useDeepLinkHandler() {
       const [url, path] = fullUrl.split("/");
       switch (url) {
         case "accounts":
+          const { address } = query;
+          if (address && typeof address === "string") {
+            const account = accounts.find(acc => acc.freshAddress === address)
+            if (account) {
+              if (account.type === "Account") {
+                navigate(`/account/${account.id}`);
+              } else {
+                navigate(`/account/${account.parentId}/${account.id}`);
```

Comment:
this case is not possible to happen. accounts is an array of Account. on the other example it's based on bellow, getAccountsOrSubAccountsByCurrency was in used and it made more sense in that case.

Reply:
Thanks Gaetan, it's fixed ! 👍